server/systemd/novaboot-unfsd-start

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
If possible, I'd prefer not having this file at all and do all things in the `.service` file.